collective/googlenews/interfaces.py

```diff
@@ -1,20 +1,72 @@
 # -*- coding: utf-8 -*-
-
+from collective.googlenews import _
+from PIL import Image
+from plone.directives import form
+from plone.formwidget.namedfile.converter import b64decode_file
+from z3c.form import field
+from z3c.form import group
 from zope import schema
 from zope.interface import Interface
+from zope.interface import Invalid
+
+import cStringIO
+import os
 
-from collective.googlenews import _
 
 TYPES_VOCAB = u"plone.app.vocabularies.ReallyUserFriendlyTypes"
 keywords_msgid = _(u"Please read https://support.google.com/news/publisher/answer/116037").replace(" ", "")
 
 
+def validate_logo(value):
+    """Valid Google news logo should follow these requirements:
```

Comment:
@rodfersou this requirements should be visible to the end user when adding the logo; add this to the field's description.